### PR TITLE
Added divide function and test

### PR DESCRIPTION
**Pull request by amyrlam**

### CODE REVIEW

File: training/js/__tests__/divide.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const divide = require('../divide');
+
+test('divide 12 / 4 to equal 3', () => {
+  expect(divide(12, 4)).toBe(3);
+});
```
Comment (yangshun) [inline training/js/__tests__/divide.test.js]: Try to configure your editor to add newlines to the end of file on save.

Comment (amyrlam) [reply training/js/__tests__/divide.test.js]: Ah, did not realize before needed a default one, in the event there's no .editorconfig in the repo. Just created an .editorconfig matching the example file here http://editorconfig.org/ in the parent directory where I keep my code. Amended

Comment (yangshun) [reply training/js/__tests__/divide.test.js]: Nice! 👍

**Comment by yangshun** [comment]
Since we want to keep the mistakes in the tests for other participants to use, we wouldn't be merging this PR. Hope the practice was good nevertheless 😄

**Pull request closed without merging**